src/image.js

```diff
@@ -47,11 +47,14 @@ export default class Image {
             .toBuffer()
             .then((part) => {
               resolve({
+                success: true,
                 position: { top: Math.round(sy), left: Math.round(sx) },
                 data: part,
               });
-            });
-        });
+            })
+            .catch(() => resolve({ success: false }));
+        })
+        .catch(() => resolve({ success: false }));
```

Comment:
This is to avoid errors when tiles are missing. It just resolves with success set to false to indicate that the tile wasn't available.